ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -192,3 +192,33 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=st.one_of(st.integers(min_value=0), st.none()),
```

Comment:
`min_num_dims` is an argument to our helper function `dtype_values_axis` which expects this argument to be an integer just like you've passed in `max_num_dims`